### ISSUE TITLE
Error installing debian packages

### ISSUE THREAD

**Issue by shaheemazmalmmd** [fossology/fossology#1194]
#### How to reproduce

generate the debian packages and install them

#### Screenshots

![error_package_install](https://user-images.githubusercontent.com/10155976/44832469-93acd380-ac48-11e8-85bd-ec995455a45c.PNG)

### Versions

* Last commit id on master: 4f77d69b1ace2e864e3bba65243ae9a6e4533b1a
* Operating System (lsb_release -a):
```
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 16.04.5 LTS
Release:	16.04
Codename:	xenial
```
### Logs
```
Reading package lists... Done
Building dependency tree       
Reading state information... Done
0 upgraded, 0 newly installed, 0 to remove and 17 not upgraded.
1 not fully installed or removed.
After this operation, 0 B of additional disk space will be used.
Setting up fossology (3.3.0-1) ...
dpkg: error processing package fossology (--configure):
 subprocess installed post-installation script returned error exit status 1
Errors were encountered while processing:
 fossology
E: Sub-process /usr/bin/dpkg returned an error code (1)
```

**Comment by GMishx** [comment]
Fixed by #1195

**Comment by GMishx** [comment]
Closed by mistake, reopening.
